File: gpt/nivel basico/go/suma.go
package main

import (
    "bufio"
    "fmt"
    "os"
    "strconv"
    "strings"
)

func main() {
    fmt.Println("Ingrese números separados por espacio:")
    scanner := bufio.NewScanner(os.Stdin)
    scanner.Scan()
    entrada := strings.Split(scanner.Text(), " ")
    suma := 0
    for _, val := range entrada {
        num, _ := strconv.Atoi(val)
        suma += num
    }
    fmt.Println("Suma:", suma)
}

File: gpt/nivel basico/c++/suma.cpp
#include <iostream>
#include <sstream>
using namespace std;

int main() {
    cout << "Ingrese numeros separados por espacio: " << endl;
    string linea;
    getline(cin, linea);
    stringstream ss(linea);
    int num, suma = 0;
    while (ss >> num) suma += num;
    cout << "Suma: " << suma << endl;
    return 0;
}
